releasenotes/notes/secret-help-add-line-break-trailing-68eab7f9895cc8e9.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    The `secret-helper` command has a new option `--remove-trailing-linebreaks` to remove any trailing linebreaks from
+    secrets read. This can be helpful when your secrets management tool automatically add an extra line break to the
+    file created.
```

Comment:
```suggestion
    The `secret-helper` command has a new option `--remove-trailing-linebreaks` to remove trailing linebreaks from
    secrets read. This can be helpful when your secrets management tool automatically adds an extra line break to the
    file created.
```